### PR TITLE
Update backlog page

### PR DESCRIPTION
**Pull request by christopherthomasdesign**
This PR updates the backlog page to:
* rename it 'Upcoming components and patterns'
* show what we're working on and what we've prioritised to work on next
* link out to a re-structured backlog repo that hosts discussion about all the other non-prioritised things

The idea is that we use the website to communicate what we're working on and what we're working on next. And we're removing the statuses / project stages from the backlog and re-framing it as a discussion space.

Reviewers should review the page itself, but also look at the content for the new [Github discussions about our 3 priorities](https://github.com/alphagov/govuk-design-system-backlog/discussions), and the [re-structured backlog](https://github.com/orgs/alphagov/projects/43/views/1). We'll be sending out comms to notify the community that this has changed.

When we publish the new page, we'll need to change the new backlog project board's permissions – it's currently private.

Part of https://github.com/alphagov/govuk-design-system/issues/2310

### CODE REVIEW

File: src/community/upcoming-components-patterns/index.md.njk [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+          }
+        ]
+      }
+    }
+  ]
+}) }}
+
+## Next priorities
+
+We particularly welcome input on the following themes. To contribute, add designs, code or research findings to the discussion on GitHub.
```
Comment (calvin-lau-sig7) [inline src/community/upcoming-components-patterns/index.md.njk:96]:
```suggestion
We particularly welcome input on the following themes. To contribute, you can add designs, code or research findings to the discussion on GitHub.
```

File: src/community/upcoming-components-patterns/index.md.njk [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      }
+    }  
+  ]
+}) }}
+
+We’ll next review the list of priorities in summer 2023. We’re not committing to publishing all the things on the list before then, but we’ll choose them first when starting new work.
+
+## Other components and patterns
+
+Browse our [GitHub discussion space](https://github.com/orgs/alphagov/projects/43/views/1) to find patterns and components that other teams have shared. Anything you can share would also be very valuable for other teams in government looking for guidance or inspiration.
```
Comment (calvin-lau-sig7) [inline src/community/upcoming-components-patterns/index.md.njk:160]:
```suggestion
Browse our [GitHub discussion space](https://github.com/orgs/alphagov/projects/43/views/1) to find patterns and components that other teams have shared. Any examples, use cases and research you can share would be very valuable for other teams in government looking for guidance or inspiration.
```

**Comment by 36degrees** [comment]
> Reviewers should review the page itself, but also look at the content for the new [Github discussions about our 3 priorities](https://github.com/alphagov/govuk-design-system-backlog/discussions), and the [re-structured backlog](https://github.com/orgs/alphagov/projects/43/views/1). We'll be sending out comms to notify the community that this has changed.

A couple of thoughts on the discussions:

- Have we considered moving the discussions into the Design System repo? Thinking if we wanted to use discussions for something else (like sharing updates, technical proposals etc) it might make sense to have everything in one place and use categories to separate them.
- It looks like we're planning on potentially having multiple discussions relating to the same component which I am very on board with. Could we use labels to group them together and to improve discoverability? (We'd need to create new labels for the prioritised components)

On the project board:

- It'd make more sense to use the **Status** field to separate components into published / not published, rather than the label.
- We could also add a new 'Type' field to split things into components, patterns and styles – the advantage of this over labels is that the project knows that these concepts are related to each other, so you can e.g. group things by their type, which you can't do with labels.

**Comment by christopherthomasdesign** [comment]
Thanks for looking @36degrees, these are all good suggestions! Will have a play around with it today. I hadn't considered putting discussions in the Design System repo, does kinda make sense.

**Review by owenatgov — COMMENTED** [review]
I can't see any technical problems with this, seems pretty clean to me 👍🏻 

I would like to clean up the commit history if possible. I haven't studied what could be grouped together just yet but I think there are opportunities to have a neater change history. Happy to help with this.

**Comment by christopherthomasdesign** [comment]
Thanks @owenatgov – I was just gonna squash it all into one commit, so if you think there are things in there worth preserving, would deffo appreciate your help

**Comment by owenatgov** [comment]
@christopherthomasdesign If you're happy to squash it into one then that's fine by me! It's not a significant enough change that it _needs_ to be split. Go ahead 😃

**Comment by trang-erskine** [comment]
Just one thing, otherwise it looks great. 

Right now, we link to the community backlog page at the start of the discussion. However, is there a way to link the discussion to the community backlog pages? Some people might be aware of our existing community backlog pages, but not about the new changes. It might be good to write a little update in the community backlog to say we are actively looking at these components right now, join the chat in the discussion pages. 

This does mean that past conversations are on the community backlog and new discussion on the discussion pages.  Which is probably okay for now. Later on we will need to think about whether we want to do anything about it.

File: src/community/upcoming-components-patterns/index.md.njk [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      }
+    }  
+  ]
+}) }}
+
+We’ll next review the list of priorities in summer 2023. We’re not committing to publishing all the things on the list before then, but we’ll choose them first when starting new work.
+
+## Other components and patterns
+
+Browse our [GitHub discussion space](https://github.com/orgs/alphagov/projects/43/views/1) to find patterns and components that other teams have shared. Any examples, use cases and research you can share would be very valuable for other teams in government looking for guidance or inspiration.
```
Comment (calvin-lau-sig7) [inline src/community/upcoming-components-patterns/index.md.njk:160]:
```suggestion
Browse our [list of discussions on GitHub](https://github.com/orgs/alphagov/projects/43/views/1) to find patterns and components that other teams have shared. Any examples, use cases and research you can share would be very valuable for other teams in government looking for guidance or inspiration.
```

**Comment by calvin-lau-sig7** [comment]
@trang-erskine, that's a good point!

We've added some [posts in the comms plan](https://docs.google.com/document/d/1RGigNkYEaS1k8LZf4SDD-vA0d8Z8HUdKK1V3zGPIs1M/edit?usp=sharing) to direct people from existing backlog discussions to the new discussion pages.

We considered whether we'd want to lock the old issue pages whilst there's a discussion happening, but agreed to wait and see what the conversations look like.

**Review by owenatgov — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**